sql/moz-fx-data-shared-prod/apple_ads_external/ad_group_report_v1/query.sql

```diff
@@ -11,8 +11,8 @@ SELECT
   start_at,
   end_at,
   taps,
-  new_downloads,
-  redownloads,
+  tap_new_downloads AS new_downloads,
+  tap_redownloads AS redownloads,
```

Comment:
I thought maybe the the new columns should be coalesced with the old columns to cover historical records, but it looks like [the dbt models already do that](https://github.com/fivetran/dbt_apple_search_ads/blob/ba2f18e161fb904c192a7268ed69647999032920/models/staging/stg_apple_search_ads__ad_group_report.sql#L39-L41), so there's no need to duplicate that logic here.